types/d3-sankey/d3-sankey-tests.ts

```diff
@@ -18,16 +18,16 @@ import { Link } from 'd3-shape';
 // the Sankey layout generator. The latter are reflected in the SankeyNode and SankeyLink interfaces provided
 // by the definitions file
 interface SNodeExtra {
-  name: string;
+    name: string;
```

Comment:
I ran prettier as per the instructions of submitting a PR and it changed a lot of the spacing